### PR TITLE
Newbr

### PR DESCRIPTION
**Pull request by feereel**
Added gitignore

### CODE REVIEW

**Comment by stancej** [comment]
- [ ] Write the code
- [ ] Write all the tests
- [ ] Document the code
- [x] My cooment

**Comment by feereel** [comment]
- [X] Write the code
- [ ] Write all the tests
- [x] Document the code

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*.c
```
Comment (stancej) [inline .gitignore:1]: Not a c files you need ignore *.py files

**Review by stancej — CHANGES_REQUESTED** [review]
- [X] Change gitignore
- [ ] Add happy file

**Comment by feereel** [comment]
- [ ] Write the code
- [ ] Write all the tests
- [ ] Document the code

**Comment by stancej** [comment]
in #1 you done in better

**Comment by feereel** [comment]
ok

**Pull request merged**